service/routes.py

```diff
@@ -73,7 +79,28 @@ def create_product():
     return (jsonify(product.serialize()), status.HTTP_201_CREATED)
 
 
-# todo - {"Location": location_url} << we have to addd this later
+# todo - {"Location": location_url} << we have to add this later
+
+
+############################################################
+# Update counters
+############################################################
+@app.route("/products/<name>", methods=["PUT"])
+def update_product(name):
+    """Update a counter"""
+    app.logger.info("Request to Update counter: '%s'...", name)
+
```

Comment:
```suggestion
check_content_type("application/json")  # Ensure correct content type
```
In Lab Flask TDD the professor asked us to make sure content type is JSON. Please fix it